### PR TITLE
Fix GitHub action deprecation warning during plugin deployment

### LINKED ISSUE #561: Investigate unexpected input `Warning` message during release build process
## Bug Description

During the release process for version 1.6.0 of the plugin the "New Release" step threw a warning: 

Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: actions/checkout, softprops/action-gh-release, actions/checkout

The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

GB action URL: https://github.com/WordPress/performance/actions/runs/3267138575

## Steps to reproduce

1. Run a release build

## Screenshots
Screenshot 2022-10-18 at 9 56 22 AM

### PR DESCRIPTION
**Pull request by mehulkaklotar**
## Summary

Fixes #561 

## Relevant technical choices

`actions/checkout@v3`
`softprops/action-gh-release@v1`
`set-output` command updated as per [GH changelog](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/)
`styfle/cancel-workflow-action@0.9.1`
`actions/cache@v3`

## Checklist

- [x] PR has either `[Focus]` or `Infrastructure` label.
- [x] PR has a `[Type]` label.
- [x] PR has a milestone or the `no milestone` label.

### CODE REVIEW

**Review by felixarntz — APPROVED** [review]
Thanks @mehulkaklotar, looks great!

Thank you for providing context in the description and the relevant link to the changelog, that is very helpful for reviewing!

**Review by mukeshpanchal27 — APPROVED** [review]
Thanks @mehulkaklotar LGTM.

**Comment by bhrajesh97** [comment]
Annotations
1 warning
🎉 Deploy
Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: SamKirkland/FTP-Deploy-Action

**Comment by bhrajesh97** [comment]
can you help me how to resolve

**Comment by mehulkaklotar** [comment]
Hi @bhrajesh97 The GH action that you are using is not updated to use Node 16. There is a [PR](https://github.com/SamKirkland/FTP-Deploy-Action/pull/329) which is not merged yet which should fix your issue when you update to use that action to latest version.

**Comment by bhrajesh97** [comment]
can you guide me how to do

**Comment by mehulkaklotar** [comment]
@bhrajesh97 you will need to wait for that PR to merge. Once it is merged, you can update your action.

**Pull request merged**
